PerimeterXModule/Internals/CredentialsIntelligence/LoginSuccessful/BodyLoginSuccessfulParser.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Web;
+using System.Text.RegularExpressions;
+
+namespace PerimeterX
+{
+    public class BodyLoginSuccessfulParser : ILoginSuccessfulParser
+    {
+        private readonly string bodyRegex;
+     
+        public BodyLoginSuccessfulParser(PxModuleConfigurationSection config) {
+            bodyRegex = config.LoginSuccessfulBodyRegex;
+        }
+
+        public bool IsLoginSuccessful(HttpResponse httpResponse)
+        {
+            HttpResponse tempHttpResponse = httpResponse;
```

Comment:
Why `tempHttpResponse`?

Reply:
Because we only can read the response and request body once.